ufo2ir/src/toir.rs

```diff
@@ -0,0 +1,46 @@
+use ir::ir;
+use norad::designspace::{self, DesignSpaceDocument};
+
+use crate::error::UfoToIrError;
+
+pub fn designspace_to_ir(designspace: DesignSpaceDocument) -> Result<Vec<ir::Axis>, UfoToIrError> {
```

Comment:
So just an observation, but I could imagine using a trait to encapsulate "convertibility to IR".